specifyweb/frontend/js_src/lib/components/Forms/SubView.tsx

```diff
@@ -204,7 +204,11 @@ export function SubView({
         <Button.BorderedGray
           aria-label={relationship.label}
           aria-pressed={isOpen}
-          className="w-fit"
+          className={`w-fit ${
```

Comment:
Also, instead of hardcoding orange color, could you use the brand color that user specified in user preferences?
And you might not need `/75` in your bg class name after that